extensions/import/src/wizard/pages/fileConfigPage.ts

```diff
@@ -340,7 +340,14 @@ export class FileConfigPage extends ImportPage {
 		let connectionUri = await sqlops.connection.getUriForConnection(this.model.server.connectionId);
 		let queryProvider = sqlops.dataprotocol.getProvider<sqlops.QueryProvider>(this.model.server.providerName, sqlops.DataProviderType.QueryProvider);
 
-		let query = `SELECT name FROM sys.schemas`;
+		let query = '';
+		if (this.databaseDropdown.value != undefined)
```

Comment:
This should be `!==` instead of `!=` since we prefer using triple equals instead of double equals